nmigen_zynq/plat.py

```diff
@@ -3,7 +3,7 @@
 
 class ZynqMPPlatform(XilinxUltraScalePlatform):
     file_templates = {
```

Comment:
Wouldn't it make more sense to do something like this?

```python
    _vivado_file_templates = {
        **XilinxUltraScalePlatform._vivado_file_templates,
```
Today there is no other toolchain than Vivado for this, but who knows...